src/vde_backend/preprocessFluksoSensors.py

```diff
@@ -325,19 +318,19 @@ def processConfig(cassandra_session, config_file_path, new_config_df, now):
 	"""
 
 	# first, create tables if necessary (if they do not already exist)
-	createTableSensorConfig(cassandra_session, "sensors_config")
-	createTableAccess(cassandra_session, "access")
-	createTableGroup(cassandra_session, "group")
+	createTableSensorConfig("sensors_config")
```

Comment:
This should use the table names defined in `constants.py` instead of hardcoded values.